src/react-vite/03-intro-to-jsx/intro-to-jsx.md

```diff
@@ -55,62 +68,216 @@ Take the below HTML and convert it to JSX:
 
 ### Setup
 
-TODO
+We've created an assets package with images and CSS you'll need to build the application. Install the `place-my-order-assets` package.
 
 ```shell
 npm install place-my-order-assets
 ```
 
-Remove contents of `src/App.css` file:
+Remove contents of `src/App.css` file and replace with the following:
 
 @diff ../../../exercises/react-vite/02-setting-up-your-environment/solution/src/App.css ../../../exercises/react-vite/03-intro-to-jsx/01-problem/src/App.css only
 
 @diff ../../../exercises/react-vite/02-setting-up-your-environment/solution/src/index.css ../../../exercises/react-vite/03-intro-to-jsx/01-problem/src/index.css only
 
+Next, you'll need to pull the image from the assets package by adding the following import into `src/App.tsx`.
+
+```tsx
+import HeroImage from 'place-my-order-assets/images/homepage-hero.jpg'
+```
+
+If you inspect the type on the import, you'll notice it is a string. It's actually just the URL for the image. You can use the image import by setting it as the `src` of an image element like `src={HeroImage}`
+
 ### Verify
 
-TODO
+You should have already installed Vitest in the previous section. If you haven't done so already, create a test file at `src/App.test.tsx` and copy the following tests into your test file.
 
 @diff ../../../exercises/react-vite/02-setting-up-your-environment/solution/src/App.test.tsx ../../../exercises/react-vite/03-intro-to-jsx/01-problem/src/App.test.tsx only
 
+Run your tests and verify the results
+
+```shell
+npm run test
+```
+
+Note that we won't be able to write an automated test that verifies the desired styles and images appear as desired, so you will have to test those manually.
+
 ### Solution
 
 <details>
 <summary>Click to see the solution</summary>
 
-TODO
+✏️ Update **src/App.tsx**
 
 @diff ../../../exercises/react-vite/03-intro-to-jsx/01-problem/src/App.tsx ../../../exercises/react-vite/03-intro-to-jsx/01-solution/src/App.tsx only
 
 </details>
 
-## Objective 2
-
-TODO
+## Objective 2 - Loops in JSX
 
 ### Key concepts
 
-TODO
+- JSX Interpolation
+- Loops in JSX
 
-#### Concept 1
+## JSX Interpolation
 
-TODO
+JSX is dynamic. You can easily insert values from variables and objects into your JSX as we did with the image URL in the previous section.
```

Comment:
I would avoid terms like “easily” because they’re beginners and it’s all new to them, so it’s not necessarily easy! In this case, I would ~~just~~ remove the word (I think words like just, obviously, simply, etc. also fall under this).